lib/scm-data-generators/git.js

```diff
@@ -0,0 +1,27 @@
+var CoreObject  = require('core-object');
+var Promise     = require('ember-cli/lib/ext/promise');
+var gitRepoInfo = require('git-repo-info');
+var simpleGit = require('simple-git');
+
+module.exports = CoreObject.extend({
+  init: function(path) {
+    this.path = path;
+  },
+
+  generate: function() {
+    var _this = this;
+    return new Promise(function(resolve, reject) {
+      simpleGit(_this.path).log(function(err, log) {
+        var info = log.latest;
+        var sha = info.hash.slice(0, 7);
+        resolve({
+          sha: info.hash.substring(1),
+          email: info.author_email.substring(0, info.author_email.length - 1),
```

Comment:
`git-simple` seems to return a traling `'` for email and a start `'` for the hash..